### ISSUE TITLE
check tag do check html balance on top of tag balance

### ISSUE THREAD

**Issue by xdecock** [jcberquist/commandbox-cfformat#80]
```cfml
<cfif test()>
<div class="wrapper">
</cfif>
<cfif test()>
</div>
</cfif>
```

This will fail the check-tag to be honnest no certain how to fix this.

**Comment by jcberquist** [comment]
@xdecock Yeah, I knew when I created this command that code like that would be an issue for it. I have such structures in my own code as well. But I don't know that there is a way to fix it, given how CFML and HTML can be intermixed. It occurred to me to just check CFML and HTML tags separately, but then the following structure will not be flagged:
```
<cfoutput>
<div>
</cfoutput>
</div>
```
I would like to catch situations like this (even though I know it is not a "real" issue) - whatever direction the tool goes, it is going to be imperfect.

**Comment by xdecock** [comment]
Maybe, list "orphan nodes" and allow them to match if they're at the same depth?

**Comment by KrunchMuffin** [comment]
I just found cfformat and started using it and I am running into this as well.
